docs/topics/settings.rst

```diff
@@ -1146,6 +1146,15 @@ Default: ``'queuelib.PriorityQueue'``
 
 Type of priority queue used by scheduler.
 
+.. setting:: SCRAPER_SLOT_MAX_ACTIVE_SIZE
+
+SCRAPER_SLOT_MAX_ACTIVE_SIZE
+----------------------------
+Default: ``5000000``
+
+Maximum size (in bytes) which the scraper can handle at once. Change this setting
+if your requests will have a body larger than this amount.
```

Comment:
The explanation about the setting is not accurate, even if the value is 1 byte the scraper will process bigger responses. 

It is a soft limit, in the sense that it process multiple responses until the sum of all responses being concurrently processed by scraper component (spider code mainly) goes above the limit. When it goes over, Scrapy core pauses the downloader (active downloads continue as normal, but new ones are not started). This mechanism prevents responses bodies from blowing up process memory by keeping only a sensible number of responses in memory.

Reply:
What about this?

> Soft limit (in bytes) for response data being processed.
>
> While the sum of the sizes of all responses being processed is above this value, Scrapy does not process new requests.